.github/workflows/package.yml

```diff
@@ -0,0 +1,97 @@
+      - run: cd agents/tracer && npm install && npm run build
+      - run: |
+          LDFLAGS="-L/opt/x-tools/$XTOOLS_HOST/$XTOOLS_HOST/sysroot/usr/lib" CPPFLAGS="-I/opt/x-tools/$XTOOLS_HOST/$XTOOLS_HOST/sysroot/usr/include/python3.10" python3 -m pip install tree_sitter
+      - run: |
+          case ${{ matrix.arch }} in
+            x86)
+              py_arch=i686
+              ;;
+            armhf)
+              py_arch=armv7l
+              ;;
+            arm64)
+              py_arch=aarch64
+              ;;
+            *)
+              py_arch=${{ matrix.arch }}
+              ;;
+          esac
+          case ${{ matrix.arch }} in
+            x86*)
+              py_platform=manylinux_2_5_${py_arch}.manylinux1_${py_arch}
+              ;;
+            arm*|ppc*|s390x)
+              py_platform=manylinux_2_17_${py_arch}.manylinux2014_${py_arch}
+              ;;
+            *)
+              py_platform=manylinux_2_5_${py_arch}
+              ;;
+          esac
+          bdist_wheel_py=/usr/lib/python3/dist-packages/wheel/bdist_wheel.py
+          sed "s/plat_name = plat_name\\.lower()\\.replace('-', '_')\\.replace('.', '_')/plat_name = plat_name.lower().replace('-', '_')/" \
+              $bdist_wheel_py > $bdist_wheel_py.patched
+          if cmp -s $bdist_wheel_py $bdist_wheel_py.patched; then
+            rm -f $bdist_wheel_py.patched
+            echo 'Unable to patch bdist_wheel.py' > /dev/stderr
+            exit 1
+          else
+            mv $bdist_wheel_py.patched $bdist_wheel_py
+          fi
+          export _PYTHON_HOST_PLATFORM=linux-$py_arch
+          echo "plat_name = $py_platform" >> setup.cfg
+          pip wheel -w ./dist --no-deps .
+      - uses: actions/upload-artifact@v3
+        with:
+          name: frida_tools_linux
+          path: dist
+
+  macos_wheels:
+    runs-on: macos-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - run: cd agents/tracer && npm install && npm run build
+      - run: python3 -m pip install wheel
+      - run: python3 -m pip install tree_sitter
+      - run: python3 setup.py bdist_wheel --plat-name=macosx_10_9_x86_64
+      - run: python3 setup.py bdist_wheel --plat-name=macosx_11_0_x86_64
+      - uses: actions/upload-artifact@v3
+        with:
+          name: frida_tools_macos
+          path: dist
+
+  windows_wheels:
+    runs-on: windows-latest
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - uses: actions/setup-python@v4
+        with:
+          python-version: '3.10'
+      - run: cd agents/tracer && npm install && npm run build
+      - run: python3 -m pip install wheel
+      - run: python3 -m pip install tree_sitter
+      - run: python3 setup.py bdist_wheel --plat-name=win_amd64
+      - run: python3 setup.py bdist_wheel --plat-name=win32
```

Comment:
Does this actually work or does it package a 64-bit .DLL in both cases?

Reply:
Yeah, it's probably 64 bit so I will remove it, don't think it's a common platform

Reply:
Right! We shouldn't specify the platform name here, the default should be fine. But what we can do is do setup-python twice, specifying the architecture each time. So that could be a matrix with the two architectures, and that's all we need to support both 32- and 64-bit.

Reply:
Done